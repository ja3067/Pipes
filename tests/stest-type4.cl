x = 3
type(x)

x = "hello"
type(x)

x = 3.0
type(x)

x = True
type(x)

x = False
type(x)

x : str = "hello"
type(x)

x : float = 3.0
type(x)

x : bool = True
type(x)

x : bool = False
type(x)

x : str = ""
type(x)
