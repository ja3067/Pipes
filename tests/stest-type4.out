int
str
float
bool
bool
str
float
bool
bool
str
